module-2/backend/week_one.md

```diff
@@ -5,30 +5,76 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get - it retrieves informatiom
+  Post - it sends the information
+  Puts - it displays the current information
+  Delete - it deletes the information
+
 2. What is Sinatra?
+  Sinatra is a "server" implemented in Ruby that is used for writing web apps.
+
 4. What is MVC?
+    MVC(models, views, controllers) is the format that is used for writing web apps.
+
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+    Following conventions makes it easier to read the code for anyone new to the project.
+    It's an established pattern within the community.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+    Variables in the model are accessible within the view templates.
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
     erb :index
   end
   ```
+Within the view file we would call horses.count.

```

Comment:
Ha! That's probably fair, but how would we get access to `horses` in the view? You can't access instance or local variables in your erb without first passing them from your controller. This is what we were looking for:

```ruby
get '/horses' do
  @count = 1
  erb :index, locals: {name: "Mr. Ed"}
end
```